docs/src/components/base/borders/borders--2px.html

```diff
@@ -0,0 +1,7 @@
+{% for color in colors %}
```

Comment:
Well, this I'm doing to properly compose fractal variants. If you name it without the `--` it won't appear as a variant